scss/grouparoo.scss

```diff
@@ -6,6 +6,9 @@
 @import "~@fortawesome/fontawesome-svg-core/styles.css";
 @import "../public/css/prism";
 
+$mobile-display-breakpoint: 600px;
+$mobile-font-breakpoint: 800px;
```

Comment:
On tablets and phones, we want to decrease the font sizes a bit

Before
<img width="839" alt="Screen Shot 2021-12-20 at 11 15 59 AM" src="https://user-images.githubusercontent.com/303226/146821770-faf1addc-8dba-4003-b677-d1b0b72da0b6.png">


After
<img width="878" alt="Screen Shot 2021-12-20 at 11 16 52 AM" src="https://user-images.githubusercontent.com/303226/146821782-3d127466-f5eb-4130-9ebf-1afde15eae47.png">

Reply:
We might want to investigate using `rem` over `px`, but I don't know if it would help handle this use-case.

Reply:
I *think* we would have the same problem - we want a bigger font on desktop than mobile, even defined relatively (rem)